packages/babel-parser/test/fixtures/es2015/arrow-functions/.inner-parens-object-pattern/input.js

```diff
@@ -0,0 +1 @@
+({ a: (foo) } = {}) => {}
```

Comment:
This test is skipped because we lost the `(foo)` error on casting Assignment Expression`{ a: (foo) } = {}` to Assignment Pattern. The `main` branch does not throw either.

Reply:
Addressed in 00a776c.